src/azure-cli/azure/cli/command_modules/vm/_vm_utils.py

```diff
@@ -105,9 +105,9 @@ def check_existence(cli_ctx, value, resource_group, provider_namespace, resource
         return False
 
 
-def create_keyvault_data_plane_client(cli_ctx):
-    from azure.cli.command_modules.keyvault._client_factory import keyvault_data_plane_factory
-    return keyvault_data_plane_factory(cli_ctx)
+def create_keyvault_data_plane_client(cli_ctx, vault_base_url):
+    from azure.cli.command_modules.keyvault._client_factory import data_plane_azure_keyvault_certificate_client
+    return data_plane_azure_keyvault_certificate_client(cli_ctx, {"vault_base_url": vault_base_url})
```

Comment:
This `create_keyvault_data_plane_client` functions returns a `azure.keyvault.certificates.CertificateClient`, making the name of this function no longer valid.

Reply:
> This `create_keyvault_data_plane_client` functions returns a `azure.keyvault.certificates.CertificateClient`, making the name of this function no longer valid.

I see. I will create a new PR to implement this change.